recipes/localdeps/meta.yaml

```diff
@@ -0,0 +1,45 @@
+{% set name = "localdeps" %}
+{% set version = "0.0.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/localdeps-{{ version }}.tar.gz
+  sha256: 62891d0854e6360a4545fe9991e30b0166f4e6ce6a5e9440afbaf61678093dfa
+
+build:
+  entry_points:
+    - localdeps = localdeps:main
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - pip
+    - python >=3.4
+  run:
+    - python >=3.4
+    - conda >=4.13
+    - conda-build >=3.22
```

Comment:
You don't have these listed upstream. Since you're also upstream, I assume that this is correct but might make sense to add them there. :)